README.md

```diff
@@ -740,6 +741,11 @@ There are a few important caveats about the provided kdump support:
 * The system kernel will reserve 256MB for the crash kernel, only when the host has at least 2GB of memory; the reserved space won't be available for processes running in the host
 * The crash kernel will only be loaded when the `crashkernel` parameter is present in the kernel's cmdline and if there is memory reserved for it
 
+### NVIDIA GPUs Support
+Bottlerocket's `nvidia` kubernetes variants include the required packages and configurations to leverage NVIDIA GPUs.
```

Comment:
nit: maybe omit `Kubernetes` since we'll have ECS soon:
```suggestion
Bottlerocket's `nvidia` variants include the required packages and configurations to leverage NVIDIA GPUs.
```